entity/src/advisory.rs

```diff
@@ -2,16 +2,22 @@ use crate::{
     advisory_vulnerability, affected_package_version_range, cvss3, fixed_package_version,
     not_affected_package_version, organization, vulnerability,
 };
+use async_graphql::*;
 use sea_orm::entity::prelude::*;
+use std::sync::Arc;
 use time::OffsetDateTime;
+use trustify_common::db;
 
-#[derive(Clone, Debug, PartialEq, Eq, DeriveEntityModel)]
+#[derive(Clone, Debug, PartialEq, Eq, DeriveEntityModel, SimpleObject)]
+#[graphql(complex)]
+#[graphql(concrete(name = "Advisory", params()))]
 #[sea_orm(table_name = "advisory")]
+
 pub struct Model {
     #[sea_orm(primary_key)]
     pub id: i32,
     pub identifier: String,
-    pub issuer_id: Option<i32>,
+    pub organization_id: Option<i32>,
```

Comment:
I think having it named 'issuer_id' is still preferred. An organization does issue it, but conceivably there could be other organizations (performing other roles) involved.

e.g., issuer, publisher, etc.

Does the GraphQL require the table name in the fk field?

Reply:
This is a common practice in GraphQL to have foreign key being labelled as "<Name of Table/Entity_id> because this quasi systematically is going to be used as the key for looking-up the record, especially in one-to-one relationship. But there are no such technical constraint for async-graphql, so we can restore the issuer_id.

Reply:
@bobmcwhirter,  I have restored `advisory.issuer_id`.